vue-app/src/store/index.ts

```diff
@@ -287,17 +287,6 @@ const actions = {
         state.currentUser.walletAddress
       )
 
-      // If the user is registered, we assume all brightId steps as done
```

Comment:
@samajammin I've separated all this brightId fetching logic into its own action (see below, LOAD_BRIGHT_ID). Why? because:
1. We have a polling made with the LOAD_USER_INFO action in the `App.vue` level and I felt that it was unnecessary to fetch brightid data all the time (unless you require to do it as in the user verification UI).
2. In the verification UI, we need to do a polling on the brightId data.

Reply:
Nice, ya I was also thinking this is probably worth breaking out separately.